src/_getFilesAncestry.ts

```diff
@@ -0,0 +1,32 @@
+import { ContentRecord } from "@activeviam/activeui-sdk-5.0";
+
+function addChildren(
+  node: ContentRecord,
+  path: { id: string; name: string }[],
+  contentRecordToPath: { [id: string]: { id: string; name: string }[] },
```

Comment:
This is mutated during the recurstion, and it's more `fileIdToPath` than `contentRecordToPath`.
So I would call it `ancestryAccumulator` or just `accumulator` since it's used only internally in this file for the recursion.

Reply:
Fair point, I'm happy with `accumulator`